### DISCUSSION TITLE
reminders  tab with days countdown

### DISCUSSION THREAD

**Discussion by leon-ITA** [usememos/memos#3949] [Ideas]
Would be useful have reminders tab with a countdown event linked to the calendar already present
i.e. i can set a date (insurance expiration date) and I can see the days left and recive a notification some days before.

**Comment by RoccoSmit** [comment]
New instance of the following related issues:
https://github.com/usememos/memos/issues/3599
https://github.com/usememos/memos/issues/2797
https://github.com/usememos/memos/issues/858
